Messaging/Two Factor Authentication via SMS - Node.js/index.js

```diff
@@ -0,0 +1,80 @@
+require("dotenv").config();
+const { Messaging } = require("@signalwire/realtime-api");
+const { phone } = require("phone");
+
+const express = require("express");
+const app = express();
+const port = 3000;
+const bodyparser = require("body-parser");
+
+app.use(bodyparser.urlencoded({ extended: true }));
+app.use("/", express.static("html"));
+
+//Global variable to store challenge sessions
```

Comment:
just a style comment, can be ignored: if you want max consistency, I suggest a space between `//` and the comment